src/pages/index.page.test.tsx

```diff
@@ -278,4 +278,57 @@ describe('Page d‘accueil', () => {
 			});
 		});
 	});
+
+	describe('World Skills', () => {
+		describe('feature flip', () => {
+			it('affiche la bannière si le feature flip est activé', () => {
+				// Given
+				process.env.NEXT_PUBLIC_WORLD_SKILLS_FEATURE = '1';
+
+				// When
+				render(
+					<DependenciesProvider analyticsService={analyticsService}>
+						<Accueil/>
+					</DependenciesProvider>,
+				);
+
+				// Then
+				expect(screen.getByText('WorldSkills Lyon 2024, la Compétition Mondiale des Métiers')).toBeInTheDocument();
```

Comment:
```suggestion
				expect(screen.getByText('WorldSkills Lyon 2024, la Compétition Mondiale des Métiers')).toBeVisible();
```
Ça va tester non-seulement que c'est dans le DOM, mais aussi que ça n'a pas d'attribut `hidden` ou autre qui causerait qu'il soit masqué (attention tout de même, ça ne testera pas le CSS custom, donc si c'est masqué par du CSS, il le considèrera quand même visible)